### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by frankkanis**
Prepared some text for translation.

And updated German translation file.

### CODE REVIEW

**Comment by Rokt33r** [comment]
@SiSchae Could you review this?

**Comment by SiSchae** [comment]
I checked it and it looks good.

File: locales/de.json [hunk 1]
```diff
@@ -41,34 +41,34 @@
...
+	"LaTeX Inline Open Delimiter": "LaTeX Inline Beginn Kennzeichen",
+	"LaTeX Inline Close Delimiter": "LaTeX Inline Ende Kennzeichen",
+	"LaTeX Block Open Delimiter": "LaTeX Block Beginn Kennzeichen",
+	"LaTeX Block Close Delimiter": "LaTeX Block Ende Kennzeichen",
 	"Community": "Community",
 	"Subscribe to Newsletter": "Newsletter abonnieren",
 	"GitHub": "GitHub",
 	"Blog": "Blog",
-	"Facebook Group": "Facebook Group",
+	"Facebook Group": "Facebook Gruppe",
```
Comment (PascalAOMS) [inline locales/de.json]: `Facebook-Gruppe` would be better. :)

File: locales/de.json [hunk 2]
```diff
@@ -77,42 +77,42 @@
...
-	"we would like to invest more time and resources in this project.": "we would like to invest more time and resources in this project.",
-	"If you like this project and see its potential, you can help by supporting us on OpenCollective!": "If you like this project and see its potential, you can help by supporting us on OpenCollective!",
-	"Thanks,": "Thanks,",
-	"Boostnote maintainers": "Boostnote Betreuer",
-	"Support via OpenCollective": "Support per OpenCollective",
+	"Dear everyone,": "Hallo,",
+	"Thank you for using Boostnote!": "Danke, dass du Boostnote verwendest.",
+	"Boostnote is used in about 200 different countries and regions by an awesome community of developers.": "Boostnote wird in über 200 verschiedenen Ländern von einer großartigen Community von Entwicklern verwendet.",
+	"To continue supporting this growth, and to satisfy community expectations,": "Um die Erwartungen der Community weiterhin erfüllen zu können und die Verbreitung von Boostnote weiter voranzutreiben,",
+	"we would like to invest more time and resources in this project.": "würden wir gern mehr Zeit und Resourcen in dieses Projekt investieren.",
```
Comment (PascalAOMS) [inline locales/de.json]: ~Resourcen~ Ressourcen

File: locales/de.json [hunk 3]
```diff
@@ -77,42 +77,42 @@
...
+	"Support via OpenCollective": "Unterstützen mit OpenCollective",
 	"Language": "Sprache",
 	"English": "Englisch",
 	"German": "Deutsch",
 	"French": "Französisch",
-	"Show \"Saved to Clipboard\" notification when copying": "\"Auf Clipboard gespeichert\" Benachrichtigungen beim kopieren anzeigen",
+	"Show \"Saved to Clipboard\" notification when copying": "\"Auf Clipboard gespeichert\" Benachrichtigungen beim Kopieren anzeigen",
 	"All Notes": "Alle Notizen",
-	"Starred": "Starred",
+	"Starred": "Markiert",
```
Comment (PascalAOMS) [inline locales/de.json]: How about `Favorisiert`?

File: locales/de.json [hunk 4]
```diff
@@ -77,42 +77,42 @@
...
 	" delete": " zu löschen",
 	"this folder?": "diesen Ordner?",
 	"Confirm": "Bestätigen",
 	"Cancel": "Abbrechen",
 	"Markdown Note": "Markdown Notiz",
-	"This format is for creating text documents. Checklists, code blocks and Latex blocks are available.": "Dieses Format dient der Erstellung von Text Dokumenten. Checklisten, Code-Blöcke und Latex-Blöcke sind verfügbar.",
-	"Snippet Note": "Snippet Notiz",
-	"This format is for creating code snippets. Multiple snippets can be grouped into a single note.": "Dieses Format dient der Erstellung von Code-Schnipslen. Mehrere snippets können zu einer Notiz zusammengefasst werden.",
+	"This format is for creating text documents. Checklists, code blocks and Latex blocks are available.": "Mit diesem Format kannst du einzelne Textdokumente erstellen. Dabei stehen dir Checklisten, Code- & Latex-Blöcke zur Verfügung.",
+	"Snippet Note": "Codeschnipsel",
```
Comment (PascalAOMS) [inline locales/de.json]: `Code-Schnipsel`

File: locales/de.json [hunk 5]
```diff
@@ -122,34 +122,87 @@
...
+	"Add": "Hinzufügen",
+	"Available Keys": "Verfügbare Tasten",
+	"Select Directory": "Ordner auswählen",
+	"copy": "Kopie",
+	"Create new folder": "Ordner erstellen",
+	"Folder name": "Ordnername",
+	"Create": "Erstellen",
+	"Untitled": "Neuer Ordner",
+	"Unlink Storage": "Speicherverknüpfung aufheben",
+	"Unlinking removes this linked storage from Boostnote. No data is removed, please manually delete the folder from your hard drive if needed.": "Die Verknüpfung des Speichers mit Boostnote wird entfernt. Es werden keine Daten gelöscht. Um die Daten dauerhaft zu löschen musst du den Ordner auf der Festplatte manuell entfernen.",
```
Comment (PascalAOMS) [inline locales/de.json]: `Um die Daten dauerhaft zu löschen, musst du den Ordner`
Comma is missing.

File: locales/de.json (hunk 3 above)
Comment (frankkanis) [reply locales/de.json]: This is a tricky one. We can discuss this. I looked how it is translated in well-known apps like gmail ("Markiert") or whatsapp ("Mit Stern markierte"). Maybe there are apps out there, where it is translated as "Favoriten"

Comment (PascalAOMS) [reply locales/de.json]: Oh I see. In that case I'm okay with that. :)
Nice research.

Comment (SiSchae) [reply locales/de.json]: "Favoriten" sounds good. :D

Comment (PascalAOMS) [reply locales/de.json]: **Starred** is an adjective, so the German translation should be one as well. **Favoriten** is a noun.

Furthermore you could simply want to _mark_ without it being your _favorite_.
I'm okay with **Markiert** (Marked).

**Pull request merged**